From c087a0e8e330ddfd352ed7a62d93b54a6e1a8d4d Mon Sep 17 00:00:00 2001
From: Witold Baryluk <witold.baryluk+github@gmail.com>
Date: Sun, 27 Dec 2020 18:03:51 +0000
Subject: [PATCH] Install radeontop to /usr/bin

Now as it doesn't necessarily require root, make it more discoverable to normal users.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 1924d5c..fae532f 100644
--- a/Makefile
+++ b/Makefile
@@ -117,7 +117,7 @@ trans:
 	--package-name radeontop
 
 install: all
-	$(INSTALL) -D -m755 $(bin) $(DESTDIR)/$(PREFIX)/sbin/$(bin)
+	$(INSTALL) -D -m755 $(bin) $(DESTDIR)/$(PREFIX)/bin/$(bin)
 ifeq ($(xcb), 1)
 	$(INSTALL) -D -m755 $(xcblib) $(DESTDIR)/$(PREFIX)/$(LIBDIR)/$(xcblib)
 endif
